.github/workflows/deploy_development.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy to AWS (Development)
```

Comment:
New dev deployment workflow. Triggers when merged to `development` and the Unit Tests workflow completes successfully.